llvm/lib/Target/AMDGPU/SIAnnotateControlFlow.cpp

```diff
@@ -38,6 +38,7 @@ using StackVector = SmallVector<StackEntry, 16>;
 
 class SIAnnotateControlFlow {
 private:
+  Function *F;
```

Comment:
If you add `= nullptr` below, why not these?

Reply:
This one is always initialized by the constructor, `F(&F)`